ci/meta.yml

```diff
@@ -78,3 +78,89 @@ meta:
           - name: dw-al2-ecs-ami
       params:
         DETAILED_EXITCODE: -detailed-exitcode
+    test-ecs-ami:
+      task: test-ecs-ami
+      config:
+        platform: linux
+        image_resource:
+          type: docker-image
+          source:
+            repository: ((dataworks.docker_awscli_repository))
+            version: ((dataworks.docker_awscli_version))
+            tag: ((dataworks.docker_awscli_version))
+        run:
+          path: sh
+          args:
+            - -exc
+            - |
+              export AWS_DEFAULT_REGION
+              source /assume-role
+              set -x
+              max_retries=15
```

Comment:
I think we should use a higher `max_retries` in here. Times for AWS Batch can vary wildly depending on availability of compute resources so, what seems fine now may fail for no reason another time.